code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -61,6 +60,7 @@
 		list("M40 MFHS Metal Foam Grenade", floor(scale * 6), /obj/item/explosive/grenade/metal_foam, VENDOR_ITEM_REGULAR),
 		list("Plastic Explosives", floor(scale * 3), /obj/item/explosive/plastic, VENDOR_ITEM_REGULAR),
 		list("Breaching Charge", floor(scale * 2), /obj/item/explosive/plastic/breaching_charge, VENDOR_ITEM_REGULAR),
+		list("M5510 Laser-Guided Rocket", floor(scale), /obj/item/ammo_magazine/rocket/brute, VENDOR_ITEM_REGULAR),
```

Comment:
This isn't really obvious as a BRUTE rocket at a glance imo-- also floor(scale) is just going to be 1 most of the time.